bpaotu/bpaotu/templates/bpaotu/otu_log.html

```diff
@@ -0,0 +1,35 @@
+<style>
+    td, th {
+        vertical-align: top;
+    }
+
+    table, th, td {
+        padding: 12px;
+        border: 1px solid;
+        border-collapse: collapse;
+    }
+</style>
+
+<table>
+    <thead>
+        <td><b>Ontology Name</b></td>
+        <td><b>Mismatches</b></td>
+    </thead>
+
```

Comment:
show the filename the error came from; sort this table by filename and then field name

Reply:
Fixed.